doc/manual/02-installation.md

```diff
@@ -0,0 +1,238 @@
+dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+```
+
+``` sh
+dnf install -y pgmoneta_ext
+```
+
+You need to add the `pgmoneta_ext` library for PostgreSQL in `postgrersql.conf` as well:
+
+```ini
+shared_preload_libraries = 'pgmoneta_ext'
+```
+
+And remember to restart PostgreSQL to make it work.
+
+## RHEL 9 / RockyLinux 9
+
+```
+dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-9-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+```
+
+``` sh
+dnf install -y pgmoneta_ext
+```
+
+You need to add the `pgmoneta_ext` library for PostgreSQL in `postgrersql.conf` as well:
+
+```ini
+shared_preload_libraries = 'pgmoneta_ext'
+```
+
+And remember to restart PostgreSQL to make it work.
+
+## Compiling the source
+
+We recommend using Fedora to test and run [**pgmoneta_ext**][pgmoneta_ext], but other Linux systems, FreeBSD and MacOS are also supported.
+
+[**pgmoneta_ext**][pgmoneta_ext] requires
+
+* [gcc 8+][gcc] (C17)
+* [cmake][cmake]
+* [make][make]
+* [PostgreSQL][postgresql]
+
+```sh
+dnf install git gcc cmake make postgresql-server postgresql-server-devel
+```
+
+Alternative [clang 8+](https://clang.llvm.org/) can be used.
+
+### RHEL / RockyLinux
+
+On RHEL / Rocky, before you install the required packages some additional repositories need to be enabled or installed first.
+
+First you need to install the subscription-manager
+
+``` sh
+dnf install subscription-manager
+```
+
+It is ok to disregard the registration and subscription warning.
+
+Otherwise, if you have a Red Hat corporate account (you need to specify the company/organization name in your account), you can register using
+
+``` sh
+subscription-manager register --username <your-account-email-or-login> --password <your-password> --auto-attach
+```
+
+Then install the EPEL repository,
+
+``` sh
+dnf install epel-release
+```
+
+Then to enable powertools
+
+``` sh
+# On RHEL 8 / Rocky 8
+dnf config-manager --set-enabled codeready-builder-for-rhel-8-rhui-rpms
+dnf config-manager --set-enabled powertools
+dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+
+# On RHEL 9 / Rocky 9, PowerTools is called crb (CodeReady Builder)
+dnf config-manager --set-enabled codeready-builder-for-rhel-9-rhui-rpms
+dnf config-manager --set-enabled crb
+dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+```
+
+Then use the `dnf` command for [**pgmoneta_ext**][pgmoneta_ext] to install the required packages.
+
+### FreeBSD
+
+On FreeBSD, `pkg` is used instead of `dnf` or `yum`.
+
+Use `pkg install <package name>` to install the following packages
+
+``` sh
+git gcc cmake gmake postgresql13-server postgresql13-client
+```
+
+### Build
+
+#### Release build
+
+The following commands will install [**pgmoneta_ext**][pgmoneta_ext] in the `/usr/local` hierarchy.
```

Comment:
It shouldn't be `/usr/local`, right ?

Reply:
It doesn't matter where you install the extension. Wherever you install it, the extension will always load into PostgreSQL to make it work.

Reply:
In practice, we don't need to specify the download location. Here, we are just simulating pgmoneta to ensure they install together.

Reply:
In our `CMake` file, we already use `pg_config` to determine important installation paths like `--sharedir`, `--includedir`, `--pkglibdir`, etc. These paths correspond to where PostgreSQL expects its extensions to be installed.

Here, we can remove the `-DCMAKE_INSTALL_PREFIX` option and not specify the installation location.